lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -269,6 +269,37 @@ def test_ecommerce_refund_failed_process_notification_sent(self, mock_send_notif
         assert call_args[0] == (course_entitlement.user, [1])
         assert refund_success
 
+    @httpretty.activate
+    @patch('lms.djangoapps.commerce.utils.send_notification_email_to_support', return_value=True)
+    def test_ecommerce_refund_failed_process_notification_email_sent(self, mock_send_notification):
```

Comment:
```suggestion
    def test_ecommerce_refund_failed_process_notification_email_sent(self, mocked_send_notification):
```

Reply:
I am trying to be consistent with existing Open edX tests. In the previous test, they have used `mock_send_notification` and so have I.